modules/serverless-rn-1-15-0.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-15-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.15.0
+
+[id="new-features-1.15.0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.21.0.
+* {ServerlessProductName} now uses Knative Eventing 0.21.0.
+* {ServerlessProductName} now uses Kourier 0.21.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.21.0.
+* {ServerlessProductName} now uses knative-kafka 0.21.0.
+// Should we include Knative `net-istio` 0.21.0?
```

Comment:
No, we've never previously mentioned it. Maybe double check with @markusthoemmes or QE folks.

Reply:
Nope, please don't mention net-istio at all.

Reply:
@markusthoemmes Thanks Markus.